### PR TITLE
Increased game requirements

### LINKED ISSUE #76: Game recorded without meeting the requirements
http://actionfps.com/game/?id=2015-12-25T18:26:03Z
how did this game register?
less than 10 mins played
less than 14 frags average cla

### PR DESCRIPTION
**Pull request by lucasgautheron**
Require 2 players on both teams, and more than 15 frags average for both teams
Fixes #76

### CODE REVIEW

File: game-parser/src/main/scala/acleague/enrichers/EnrichFoundGame.scala [hunk 1]
```diff
@@ -168,11 +168,11 @@ case class JsonGame(id: String, endTime: ZonedDateTime, map: String, mode: Strin
...
 
   def validate: JsonGame Or ErrorMessage = {
-    def numberOfPlayers = teams.map(_.players.size).sum
-    def averageFrags = teams.flatMap(_.players.map(_.frags)).sum / numberOfPlayers
+    def numberOfPlayers = teams.map(_.players.size).min
+    def averageFrags = teams.map(x => x.players.map(_.frags).sum.toFloat/x.players.size).min
     if (duration < 10) Bad(s"Duration is $duration, expecting at least 10")
     else if (duration > 15) Bad(s"Duration is $duration, expecting at most 15")
-    else if (numberOfPlayers < 4) Bad(s"Player count is $numberOfPlayers, expecting 4 or more.")
+    else if (numberOfPlayers < 2) Bad(s"Player count is $numberOfPlayers, expecting 4 or more.")
```
Comment (ScalaWilliam) [inline game-parser/src/main/scala/acleague/enrichers/EnrichFoundGame.scala]: Text is wrong?

File: game-parser/src/main/scala/acleague/enrichers/EnrichFoundGame.scala [hunk 2]
```diff
@@ -168,11 +168,11 @@ case class JsonGame(id: String, endTime: ZonedDateTime, map: String, mode: Strin
   import org.scalactic._
 
   def validate: JsonGame Or ErrorMessage = {
-    def numberOfPlayers = teams.map(_.players.size).sum
-    def averageFrags = teams.flatMap(_.players.map(_.frags)).sum / numberOfPlayers
+    def numberOfPlayers = teams.map(_.players.size).min
```
Comment (ScalaWilliam) [inline game-parser/src/main/scala/acleague/enrichers/EnrichFoundGame.scala]: Read the code from scratch, do the names and texts make sense with the code?

Comment (lucasgautheron) [inline game-parser/src/main/scala/acleague/enrichers/EnrichFoundGame.scala]: i thought it still made sense but you're right it should be changed. 

numberOfPlayers -> smallestTeamPlayers? 
averageFrags -> lowestAverageFrags ? 

And text changed accordingly

Comment (ScalaWilliam) [inline game-parser/src/main/scala/acleague/enrichers/EnrichFoundGame.scala]: minTeamPlayers
minTeamFrags

**Comment by ScalaWilliam** [comment]
lgtm :+1: feel free to merge

**Pull request merged**